test/char.ts

```diff
@@ -45,6 +45,7 @@ describe('char', () => {
   it('space', () => {
     const parser = space
     eqEithers(parser.run(' '), createParseSuccess(' ', ''))
+    eqEithers(parser.run('\t'), createParseSuccess('\t', ''))
```

Comment:
Add a test for `\n` too?

And perhaps test that `"\n\t "` is whitespace is `test/strings.ts`?

Reply:
I like how you offload whitespace checking to a regex

Reply:
@danr thanks, PR updated